packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -157,14 +176,15 @@ export async function generateOutputDirectory(
   if (outputBundleOptions.needsServerGenerated) {
     await generateServer(outputBundleOptions);
   }
+  if (!angularVersion) throw new Error("Could not find the angular version of the application");
   await generateBundleYaml(outputBundleOptions, cwd, angularVersion);
 }
 
 // add environment variable to bundle.yaml if needed for specific versions
-function addBundleYamlEnvVar(angularVersion?: string): EnvironmentVariable[] {
+function addBundleYamlEnvVar(angularVersion: string): EnvironmentVariable[] {
```

Comment:
naming nit: can we change this to getEnvVars or createEnvVars? This isn't really adding env vars to bundle.yaml yet

Reply:
ah i see we have `generateBundleYaml` below, how about `generateEnvVars`